docs_source_files/content/front_matter/copyright_and_attributions.de.md

```diff
@@ -40,20 +34,20 @@ to the use of the information contained herein.
 
 {{% ghcontributors "https://api.github.com/repos/seleniumhq/docker-selenium/contributors?per_page=100" %}}
 
-#### [Selenium Website & Docs](//github.com/SeleniumHQ/seleniumhq.github.io/)
+#### [Selenium Webseite & Dokumentation](//github.com/SeleniumHQ/seleniumhq.github.io/)
 
 {{% ghcontributors "https://api.github.com/repos/seleniumhq/seleniumhq.github.io/contributors?per_page=100" %}}
 
-#### [Previous Selenium Website](//github.com/SeleniumHQ/www.seleniumhq.org/)
+#### [Vorhergehende Selenium Webseite](//github.com/SeleniumHQ/www.seleniumhq.org/)
 
 {{% ghcontributors "https://api.github.com/repos/seleniumhq/www.seleniumhq.org/contributors?per_page=100" %}}
 
-#### [Previous Documentation Rewrite Project](//github.com/SeleniumHQ/docs/)
+#### [Vorgänger Projekt zur Neuentwicklung der Dokumentation](//github.com/SeleniumHQ/docs/)
 
 {{% ghcontributors "https://api.github.com/repos/seleniumhq/docs/contributors?per_page=100" %}}
 
 
-## Third-Party software used by Selenium documentation project:
+## rittanbieter Software die vom Selenium Dokumentationsprojekt genutzt wird:
```

Comment:
```suggestion
## Drittanbieter Software die vom Selenium Dokumentationsprojekt genutzt wird:
```

Reply:
you are right.

Reply:
👍 if you want to, just click on `Commit suggestion` or push a new commit with the change, then we can merge it.

Reply:
Done (with a push) changed your suggestion and did a merge from master